config/crd/bases/ssp.kubevirt.io_ssps.yaml

```diff
@@ -22,21 +22,27 @@ spec:
         description: SSP is the Schema for the ssps API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
```

Comment:
Please, do not commit these changes to the line wrap for descriptions. I don't know why on some systems the `controller-gen` generates the `description` on multiple lines and somewhere on one line.